### DISCUSSION TITLE
no module named 'mooseutils'

### DISCUSSION THREAD

**Discussion by wangzhaohao** [idaholab/moose#25424] [Q&A Tools] [answered]
i want to use the [parameter study](https://mooseframework.inl.gov/modules/stochastic_tools/examples/parameter_study.html) and got the results. But when i used Python to see the Statistics and Distribution, there report 
```
no module named 'mooseutils'
```
where to find it and how to put it in the path?
Thanks for you advice!

**Comment by GiudGiud** [comment, marked as answer]
@milljm will know

Did you try:
- adding moose/python to your PYTHONPATH
- installing moose-tools through mamba?

**Reply by wangzhaohao** [reply]
i will try first. i have install moose tools by mamba, thanks

**Reply by wangzhaohao** [reply]
i have sloved it.

**Reply by milljm** [reply]
Responding for the sake of there being an answer without detail, if someone searches for this error, and lands here.

`mooseutils` is a python module within the MOOSE repository, not relating to our Conda packages.

```pre
moose/python/mooseutils
```

For the error received one would have had to have a custom PYTHONPATH, or MOOSE_DIR being set (incorrectly, or incomplete).
